math/ecgen/Portfile

```diff
@@ -4,13 +4,13 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        J08nY ecgen 0.7.6
+github.setup        J08nY ecgen 0.7.7
 github.tarball_from archive
 revision            0
 
-checksums           rmd160  d4906f4781976d54caca8910b30fb84e76744f88 \
-                    sha256  b7ccfcc605898204652b877a3b0dca42703682471bec8e4dda46de29b66add23 \
-                    size    223697
+checksums           rmd160  0ddc48762bb47585a5a6e11ebc5c76583748fdb9 \
+                    sha256  d4d0d4066606d8582651e4789537928aec3c7e325b82f58dd2266350edf95371 \
+                    size    223674
 
 categories          math
 platforms           darwin
```

Comment:
Now that `platforms darwin` is implicit, we can begin to remove this line as we update ports.

Reply:
Indeed. Thanks!